### ISSUE TITLE
Spending multiple coins from one address in the Coins tab results in a needlessly big transaction

### ISSUE THREAD

**Issue by l29ah** [spesmilo/electrum#5818]
When i select N coins located in the same wallet in the Coins tab, the resulting transaction contains one entry per coin instead of one entry for the whole source wallet, so the transaction becomes almost N times bigger than the optimal one.

**Comment by SomberNight** [comment]
I don't understand your explanation. What is a wallet? Do you mean a bitcoin address?

**Comment by l29ah** [comment]
Yes.

**Comment by SomberNight** [comment]
Bitcoin is UTXO-based, not account-based. This is how it works. Each UTXO ("coin") needs to be enumerated as an input.

**Comment by l29ah** [comment]
Thank you!
